data/hai/hai missions.txt

```diff
@@ -2099,3 +2099,205 @@ mission "Hai Pilgrimage"
+				`	"Sure thing. I'd be glad to take you back."`
+					accept
+	on visit
+		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
+
+mission "Acorn Delights: Part 5"
+	name "Eeeya to Anoo"
+	description "Bring Eeeya to <destination> to join the Unfettered and be near her granddaughter."
+	landing
+	clearance `The Darkcloak flight controller lets you land when they hear that Eeeya is joining the Unfettered.`
+	source "Allhome"
+	destination "Darkcloak"
+	passengers 1
+	to offer
+		has "Acorn Delights: Part 4: done"
+	on offer
+		conversation
+			branch uhai
+				has "acorn delights: eeeya joined"
+			action
+				"salary: Acorn Delights" = 200
+				log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo. She ran off to join the Unfettered, but Eeeya wasn't interested in doing so. Received stocks for a two hundred credit salary for my efforts.`
+			`Eeeya immediately gets off the ship and goes to check on the acorn cookie shop. A few minutes later, she is back with documents for you to sign.`
+			`	"These will set you up with about a tenth of the stock in my company; you'll receive two hundred credits daily." As you sign the documents, she continues, "I'd just like to thank you again for helping me find Anoo. It means a lot." You check that the stocks have been transferred to your account, and Eeeya heads off.`
+				decline
+			label uhai
+			`Eeeya heads off to sell Acorn Delights' shop while you finish the landing procedure. Soon, she is back. "Well, there was a Hai who paid thirty million credits for that small place. I wonder why he thought it was worth so much - I sure don't. But I won't argue with that nutty person." She seems much merrier now that Anoo has been found. "I'll be ready to go when you are."`
+				accept
+	on complete
+		payment 1000000
+		log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo. After a long journey, we found her - she had gone to help the Unfettered as a trauma surgeon. Brought Eeeya back to <origin> so she could sell the shop, but both Hai are together on <destination> now. Received a total of one million and three thousand credits for my efforts.`
+		conversation
+			`Eeeya leaves to go to the hospital, after paying you five hundred thousand credits and thanking you for the third time. But before you are allowed to leave your ship, an Unfettered security officer arrives. "Come with me<, unfettered title>, we have something to say to you."`
+			`	"Why?" you ask. The Unfettered leads you to an office without answering. "We would like to thank you for recruiting a Hai - especially one as renowned as Chef Eeeya. You are being paid an additional five hundred thousand credits; do not think it is too much. You are dismissed." As you head back to <ship>, your communicator receives a receipt for the promised payment.`
```

Comment:
Why would the Unfettered be so thrilled that a cookie baker is joining their ranks?